src/components/CounterInput/helpers/isValueNan.js

```diff
@@ -0,0 +1,8 @@
+const isValueNan = value => {
```

Comment:
remember `is` suffix is used for booleans, this helper returns a value then it should be called like `getSomething` with the `get` suffinx